src/app/ui/home/home.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
```

Comment:
Watch the id: `id="carouselExampleIndicators"`. Change it to something the describes the actual element.

Reply:
Make sure you change the id in all referenced locations too. For example, in lines 4,5,6 and many more.